components/support/nimbus-fml/src/backends/kotlin/mod.rs

```diff
@@ -2,22 +2,28 @@
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use crate::error::Result;
+use crate::error::{FMLError, Result};
 use askama::Template;
 
 use crate::intermediate_representation::FeatureManifest;
-use crate::{Config, GenerateStructCmd};
+use crate::GenerateStructCmd;
 
 mod gen_structs;
 
-pub(crate) fn generate_struct(
-    manifest: FeatureManifest,
-    config: Config,
-    cmd: GenerateStructCmd,
-) -> Result<()> {
-    let kt = gen_structs::FeatureManifestDeclaration::new(config, &manifest);
+pub(crate) fn generate_struct(manifest: &FeatureManifest, cmd: &GenerateStructCmd) -> Result<()> {
+    if manifest.about.kotlin_about.is_none() {
+        return Err(FMLError::ValidationError(
+            "about".to_string(),
+            format!(
+                "The `about` block is missing a valid `android` entry: {}",
```

Comment:
Similar to the Swift backend, this refers to the entry as `android` but the test yaml appears to be looking for `kotlin`